mapreader/load/geo_utils.py

```diff
@@ -60,53 +60,46 @@ def reprojectGeoInfo(image_path, proj2convert="EPSG:4326", calc_size_in_m=False)
 
     # Coordinate transformation: proj1 ---> proj2
     transformer = Transformer.from_crs(tiff_proj, proj2convert)
```

Comment:
Yes definitely agree with ``target_crs`` 

Also happy to rename / split ``reproject_geo_info`` - at the moment its a slightly redundant function and not used anywhere (unless by user) so should be able to play around with it without consequences.

Reply:
`proj2convert` also appears in `images.py`, and I think it would be good to rename that to `target_crs` as well.

If `reprojectGeoInfo` is not really used, then leave it for now, but create a low-priority ticket to look at removing or refactoring this at some future date.